source/_patterns/03-uswds/button/button--accent-cool/button--accent-cool.md

```diff
@@ -0,0 +1,12 @@
+---
+el: .usa-button--accent-cool
+title: Button (Accent Cool)
```

Comment:
I don’t feel strongly which format we use, but we should be consistent. If we want to switch to “Component (Variant)” then we need to convince folks to update Gesso to match.

Reply:
I don't feel strongly about format either. I like Component (Variant) because the name order then matches the BEM class name order, but if you want to switch it to Variant Component to match other components, I'm good with that.